src/backend/glocal.c

```diff
@@ -214,6 +214,7 @@ STATIC void local_exp(elem *e,int goal)
                             em->E1->EV.sp.Vsym == s &&
                             tysize(em->Ety) == tysize(e1->Ety) &&
                             em->E1->EV.sp.Voffset == e1->EV.sp.Voffset &&
+                            (config.flags4 & CFG4fastfloat || !tyfloating(em->Ety)) &&
```

Comment:
extra parens for clarity?

    ((config.flags4 & CFG4fastfloat) || !tyfloating(em->Ety)) &&